shared.bazelrc

```diff
@@ -15,6 +15,9 @@ build --incompatible_strict_action_env=true
 common --enable_bzlmod
 build --@cgrindel_bazel_starlib//bzlmod:enabled
 
+# Set minimum macOS version
+build --macos_minimum_os=13 --host_macos_minimum_os=13
```

Comment:
If CI passes with this, I'll see if I can remove it or at least lower the minimum version.

Reply:
I was able to lower the minimum version to 10.15 and set this in `tca_example`'s `.bazelrc`.